File: FMI/ANUL II/SEM 2/MPP/ProiectJava/Zboruri/ZboruriServices/src/main/java/ro/mpp2024/services/ZboruriException.java
package ro.mpp2024.services;

public class ZboruriException extends Exception{
    public ZboruriException(){}
    public ZboruriException(String message) {
        super(message);
    }
    public ZboruriException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: FMI/ANUL II/SEM 2/MPP/ProiectCSharp/Zboruri/Services/ZboruriException.cs
namespace Services;

public class ZboruriException : Exception
{
    public ZboruriException()
    {
    }

    public ZboruriException(String message) : base(message)
    {
    }

    public ZboruriException(String message, Exception cause) : base(message, cause)
    {
    }
}